cmd/webhook.go

```diff
@@ -92,12 +96,26 @@ var webhookCmd = &cobra.Command{
 			}
 		}(c, dnsName, certCheckInterval)
 
+		cipherList, err := getTLSCipherSuitesIDs(tlsCiphers)
+		if err != nil {
+			ctrl.Log.Error(err, "unable to fetch tls ciphers")
+			os.Exit(1)
+		}
+		mgrTLSOptions := func(cfg *tls.Config) {
+			cfg.CipherSuites = cipherList
+		}
 		mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 			Scheme:                 scheme,
 			MetricsBindAddress:     metricsAddr,
 			HealthProbeBindAddress: healthzAddr,
 			Port:                   port,
 			CertDir:                certDir,
+			WebhookServer: &webhook.Server{
+				TLSMinVersion: tlsMinVersion,
+				TLSOpts: []func(*tls.Config){
+					mgrTLSOptions,
+				},
+			},
```

Comment:
I just learned that if you configure `options.WebhookServer` yourself, then `options.Port` and `options.CertDir` does not get copied to `options.WebhookServer.Port` and `options.WebhookServer.CertDir` (which makes sense). Instead, we must specify the port and certDir in the `options.WebhookServer` struct.

Could you please move these options from line 111+1112 down to the `&webhook.Server{ ... }` struct here?

Reply:
I tested it locally, this should bring the server up and e2e tests should be 🥬

Reply:
I see what you mean! Fixed, thanks!